### PR TITLE
Point documentation links to monorepo

### PR DESCRIPTION
**Pull request by skunert**
## Adjusted:
- Links in markdown files that are referencing docs in the old repos
- Links in Dockerfiles that were referencing outdated files in new or old repo
- Some Docker instructions that did not work due to new repo structure
- Mentioned pull requests were it was easy to find follow ups in the new repository

## Not adjusted:
- Links to issues in the code. When they were transferred they will point to the new one anyways. For stale links we could make another pass. But there are tons of them.
- Links to old pull requests for reference.
- Links to code files in the old repos. Ideally they would point to the new repo, but for this we should check that the description is still correct.

### CODE REVIEW

File: bridges/modules/parachains/README.md [hunk 1]
```diff
@@ -9,15 +9,15 @@ to verify storage proofs, generated at the bridged relay chain.
...
+You can find detailed information on parachains finality in the
+[Polkadot-SDK](https://github.com/paritytech/polkadot-sdk) repository. This section gives a brief overview of how the
+parachain finality works and how to build a light client for a parachain.
 
 The main thing there is that the parachain generates blocks on its own, but it can't achieve finality without
 help of its relay chain. Instead, the parachain collators create a block and hand it over to the relay chain
 validators. Validators validate the block and register the new parachain head in the
-[`Heads` map](https://github.com/paritytech/polkadot/blob/88013730166ba90745ae7c9eb3e0c1be1513c7cc/runtime/parachains/src/paras/mod.rs#L645)
-of the [`paras`](https://github.com/paritytech/polkadot/tree/master/runtime/parachains/src/paras) pallet,
+[`Heads` map](https://github.com/davxy/polkadot-sdk/blob/bc5005217a8c2e7c95b9011c96d7e619879b1200/polkadot/runtime/parachains/src/paras/mod.rs#L683-L686)
```
Comment (michalkucharczyk) [inline bridges/modules/parachains/README.md:19]:
```suggestion
[`Heads` map](https://github.com/paritytech/polkadot-sdk/blob/bc5005217a8c2e7c95b9011c96d7e619879b1200/polkadot/runtime/parachains/src/paras/mod.rs#L683-L686)
```

**Review by michalkucharczyk — APPROVED** [review]

**Review by coderobe — APPROVED** [review]

**Review by sam0x17 — APPROVED** [review]

File: docker/dockerfiles/polkadot-parachain/polkadot-parachain_builder.Containerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# This file is sourced from https://github.com/paritytech/polkadot/blob/master/docker/dockerfiles/polkadot/polkadot_builder.Dockerfile
+# This file is sourced from https://github.com/paritytech/polkadot-sdk/blob/master/docker/dockerfiles/polkadot/polkadot_builder.Dockerfile
```
Comment (chevdor) [inline docker/dockerfiles/polkadot-parachain/polkadot-parachain_builder.Containerfile:1]: The issue is not introduced by your PR but would you mind renaming that file to `.Dockerfile` ?

File: docker/dockerfiles/polkadot-parachain/polkadot-parachain_builder.Containerfile [hunk 3]
```diff
@@ -14,8 +14,8 @@ LABEL io.parity.image.type="builder" \
     io.parity.image.authors="devops-team@parity.io" \
     io.parity.image.vendor="Parity Technologies" \
     io.parity.image.description="Multistage Docker image for polkadot-parachain" \
-    io.parity.image.source="https://github.com/paritytech/polkadot/blob/${VCS_REF}/docker/dockerfiles/polkadot-parachain/polkadot-parachain_builder.Dockerfile" \
-    io.parity.image.documentation="https://github.com/paritytech/cumulus"
+    io.parity.image.source="https://github.com/paritytech/polkadot-sdk/blob/${VCS_REF}/docker/dockerfiles/polkadot-parachain/polkadot-parachain_builder.Containerfile" \
```
Comment (chevdor) [inline docker/dockerfiles/polkadot-parachain/polkadot-parachain_builder.Containerfile:17]:
```suggestion
    io.parity.image.source="https://github.com/paritytech/polkadot-sdk/blob/${VCS_REF}/docker/dockerfiles/polkadot-parachain/polkadot-parachain_builder.Dockerfile" \
```

File: docker/dockerfiles/test-parachain-collator.dockerfile [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# This file is sourced from https://github.com/paritytech/polkadot/blob/master/docker/dockerfiles/polkadot/polkadot_builder.Dockerfile
+# This file is sourced from https://github.com/paritytech/polkadot-sdk/blob/master/docker/dockerfiles/test-parachain-collator.dockerfile
```
Comment (chevdor) [inline docker/dockerfiles/test-parachain-collator.dockerfile:1]:
```suggestion
# This file is sourced from https://github.com/paritytech/polkadot-sdk/blob/master/docker/dockerfiles/test-parachain-collator.Dockerfile
```

**Review by chevdor — APPROVED** [review]
LGTM beside the Dockerfile naming issues.
Those are not due to your PR but it would be nice to fix them as well.

**Review by svyatonik — APPROVED** [review]

**Pull request merged**